server/conf/i18n/messages

```diff
@@ -97,49 +97,61 @@ title.submittedPrograms=Submitted
 toast.applicationSaved=Successfully saved application: application ID {0}
 toast.programCompleted=Application was already completed.
 
-#------------------------------------------------------------------------#
-# TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding a client #
-#------------------------------------------------------------------------#
+#------------------------------------------------------------------------------------------------------#
+# TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding, editing, deleting, or searching for a client #
+#------------------------------------------------------------------------------------------------------#
 
+banner.acctDeleteError=This account cannot be deleted.
+banner.acctDeleteErrorReason=This account cannot be deleted since it has open applications.
 banner.clientInfoUpdated=Client info has been successfully updated.
 banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
-banner.acctDeleteError=This account cannot be deleted.
+banner.newClientCreated=New client successfully created.
 
 button.addNewClient=Add new client
+button.backToClientList=Back to client list
 button.backToEditing=Back to editing
 button.cancel=Cancel
+button.clearSearch=Clear search
 button.deleteAcct=Delete account
 button.keepAcct=Keep account
 button.nextPage=Next
 button.save=Save
 button.search=Search
+button.select=Select
+button.startApp=Start an application
 button.viewApplications=View applications
 
+content.clientCreated=You new client {0} has been added to your client list. You can now begin submitting applications for this client.
 content.numberOfAppSubmitted={0} applications submitted
 content.warning=There are no applications submitted for this client, so no applications will be lost.
 
-dialog.deleteConfirmation=Are you sure you want to delete this client's account?
+dialog.deleteConfirmation=Are you sure you want to delete this client''s account?
 
-label.dob=Date of birth
-label.nameOrEmail=Name(s) or email
-label.notes=Notes
+header.acctSettings=Account settings
+header.clientList=Client list
+
+label.acctStatus=Account Status
 label.contactInfo=Contact Information
+label.day=Day
+label.dobExample=For example: January 28,1986
+label.email=Email
+label.month=Month
+label.name=Name
+label.nameExample=ex. Darren Gu, Gu or darren.gu@gmail.com
```

Comment:
suggestion (blocking):  This one was updated today too.  It should be "For example: Gu or Darren or Darren Gu".

Reply:
ah i see, it was literally `ex. Darren Gu, Gu or darren.gu@gmail.com` just one hour ago. updated.